include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -217,9 +217,9 @@ struct RunOptions : Base<OrtRunOptions> {
 };
 
 struct SessionOptions : Base<OrtSessionOptions> {
-  explicit SessionOptions(std::nullptr_t) {}
+  explicit SessionOptions(std::nullptr_t);
```

Comment:
curious - why did you land on putting this in the SessionOptions constructor and not in the Session() wrapper constructor?   what will happen is someone uses the session with a null for sessionptions?  and never constructs one?

Reply:
The reason that we don't put this in the Session() constructor is that the sessionOptions in Session Constructor is const. If someone doen't pass in a sessionoptions, from the current logics for these added languages, the a default sessionOption will be created when construct a session.